r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -18,6 +18,27 @@
 library(dplyr)
 library(stringr)
 
+test_that("sub and gsub with ignore.case = FALSE and fixed = TRUE", {
+  df <- tibble(x = c("Foo", "bar"))
```

Comment:
Do these tests require skip_if_not_available("utf8proc")?

Reply:
Yes, they do depend on utf8proc. I'll add that at the top.

Reply:
Added in da7defb53. Thanks for catching that!